pycbc/strain/strain.py

```diff
@@ -368,6 +368,10 @@ def from_cli(opt, dyn_range_fac=1, precision='single',
         logging.info("Highpass Filtering")
         strain = highpass(strain, frequency=opt.strain_high_pass)
 
+    if opt.strain_low_pass:
+        logging.info("Lowpass Filtering")
+        strain = lowpass(strain, frequency=opt.strain_low_pass)
```

Comment:
Is this also done when simulating fake noise? Should it be?

Reply:
I believe this does apply when generating fake noise - I'll check that explicitly